salesagility/SuiteCRM
# Remove call to parent constructor deprecated in 7.11

## Description
Remove call to parent constructor deprecated in 7.11

This was producing a very annoying log spam with a blank message, hard to figure out where it was coming for.

``` 
Fri Sep 22 11:42:38 2023 [11910][1][DEPRE] Array
(
)
```

## How To Test This
This blank deprecated message is probably showing in the logs all the time, but a specific place where I am seeing it (so you can test in the same place as me) is saving a Leads record.

After removing  the call to the constructor, the message should go away.


## Types of changes


### Final checklist
